docs/tutorials/excel-power-automate-returns.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Return data from a script to an automatically-run Power Automate flow
 description: A tutorial that shows how to send reminder emails by running Office Scripts for Excel through Power Automate.
+ms.topic: tutorial
```

Comment:
there's already an entry in the docfx.json so all the .md files in the tutorials folder are set to tutorial